cmd/nerdctl/create_linux_test.go

```diff
@@ -109,6 +111,32 @@ func TestCreateWithMACAddress(t *testing.T) {
 	}
 }
 
+func TestCreateDuplicateName(t *testing.T) {
```

Comment:
Probably this test is not specific to Linux, and can be moved to `create_test.go`

Reply:
Also could you add comment lines to explain this test scenario?